src/websocket/stream.py

```diff
@@ -221,53 +197,35 @@ async def transcribe_chunk_partial(
                 result = json.dumps({"partial": text}, indent=2)
             else:
                 self.logger.error("Transcription Data is empty, no segments found")
-            await websocket.send(result)
+
+            await websocket.send_text(result)
 
             end_time = time.time()
             self.logger.debug(
                 "Partial transcription took {:.2f} s".format(end_time - start_time)
             )
+
         except Exception:
             self.logger.error(
                 "Error while transcribing audio: {}".format(traceback.format_exc())
             )
             self.close_stream = True
 
     def adjust_threshold_on_latency(self):
```

Comment:
I like that you are simplifying this! However:
- What steps did you take to ensure that it still works as intended?
- Maybe keep some of the comments? Otherwise it might become hard to maintain for anyone new coming into the project

Reply:
the simplification is the exact same code just refactored, so no changes there

Good catch on the rest, I just completely removed the partial exclusion, there was no reason I remember